### PR TITLE
[docs] Fix installation instructions in c# documentation

### PR DESCRIPTION
**Pull request by kandersen82**
### Motivation

The installation instructions were for a completely unrelated package.

### Modifications

Modified the documentation to describe installation of the DotPulsar package instead of Newtonsoft.Json

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): no
  - The public API: no
  - The schema: no
  - The default values of configurations: no
  - The wire protocol: no
  - The rest endpoints: no
  - The admin cli options: no
  - Anything that affects deployment: no

### Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by RobertIndie — APPROVED** [review]
LGTM

**Review by Huanli-Meng — APPROVED** [review]
LGTM.

But, please remember to update docs in other available versions.

**Comment by kandersen82** [comment]
Updated in doc for previous versions

**Comment by Huanli-Meng** [comment]
> Updated in doc for previous versions

Thanks.

File: site2/docs/client-libraries-dotnet.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Pulsar C# client
 sidebar_label: C#
 ---
 
-You can use the Pulsar C# client to create Pulsar producers and consumers in C#. All the methods in the producer, consumer, and reader of a C# client are thread-safe.
+You can use the Pulsar C# client (DotPulsar) to create Pulsar producers and consumers in C#. All the methods in the producer, consumer, and reader of a C# client are thread-safe. The official documentation for DotPulsar is available [here](https://github.com/apache/pulsar-dotpulsar/wiki)
```
Comment (Anonymitaet) [inline site2/docs/client-libraries-dotnet.md:7]:
```suggestion
You can use the Pulsar C# client (DotPulsar) to create Pulsar producers and consumers in C#. All the methods in the producer, consumer, and reader of a C# client are thread-safe. The official documentation for DotPulsar is available [here](https://github.com/apache/pulsar-dotpulsar/wiki).
```

File: site2/docs/client-libraries-dotnet.md [hunk 2]
```diff
@@ -32,80 +32,35 @@ To install the Pulsar C# client library, following these steps:
 
    4. Use `dotnet run` to test that the app has been created properly.
 
-2. Add the Newtonsoft.Json NuGet package.
+2. Add the DotPulsar NuGet package.
 
-   1. Use the following command to install the `Newtonsoft.json` package:
+   1. Use the following command to install the `DotPulsar` package:
```
Comment (Anonymitaet) [inline site2/docs/client-libraries-dotnet.md:37]:
```suggestion
   1. Use the following command to install the `DotPulsar` package.
```

File: site2/docs/client-libraries-dotnet.md [hunk 3]
```diff
@@ -32,80 +32,35 @@ To install the Pulsar C# client library, following these steps:
...
 
         ```
-        dotnet add package Newtonsoft.Json
+        dotnet add package DotPulsar
         ```
 
    2. After the command completes, open the `.csproj` file to see the added reference:
```
Comment (Anonymitaet) [inline site2/docs/client-libraries-dotnet.md:43]:
```suggestion
   2. After the command completes, open the `.csproj` file to see the added reference.
```

File: site2/docs/client-libraries-dotnet.md [hunk 4]
```diff
@@ -32,80 +32,35 @@ To install the Pulsar C# client library, following these steps:
...
 
-This example shows how to create a Pulsar C# client connected to the local host.
+This example shows how to create a Pulsar C# client connected to localhost.
 
 ```c#
 var client = PulsarClient.Builder().Build();
 ```
 
-To create a Pulsar C# client by using the builder, you need to specify the following options:
+To create a Pulsar C# client by using the builder, you can specify the following options:
```
Comment (Anonymitaet) [inline site2/docs/client-libraries-dotnet.md:63]:
```suggestion
To create a Pulsar C# client by using the builder, you can specify the following options.
```

**Review by Anonymitaet — COMMENTED** [review]
Thanks for your contribution. I've made some comments, pls check all occurrences.

**Review by sijie — APPROVED** [review]

**Comment by kandersen82** [comment]
@sijie what is the reasoning behind placing this in milestone 2.8.0? As i see it is simply a fix for a mistake in the documentation, that should be possible to merge right away?

**Comment by blankensteiner** [comment]
> @sijie what is the reasoning behind placing this in milestone 2.8.0? As i see it is simply a fix for a mistake in the documentation, that should be possible to merge right away?

Agreed. From DotPulsar's perspective it would really be appreciated if this could be merged asap.

**Review by Jennifer88huang — APPROVED** [review]

**Comment by Jennifer88huang** [comment]
@kandersen82 @blankensteiner Your changes will take effect in 2.6.x, 2.7.x and 2.8 since you have applied those changes to the versioned docs directly, do not worry about it. 
2.8.0 is the upcoming major release, so we set it as a milestone. Since your changes should be included in the 2.8 release, we add the `milestone 2.8.0` label.

**Comment by kandersen82** [comment]
> @kandersen82 @blankensteiner Your changes will take effect in 2.6.x, 2.7.x and 2.8 since you have applied those changes to the versioned docs directly, do not worry about it.
> 2.8.0 is the upcoming major release, so we set it as a milestone. Since your changes should be included in the 2.8 release, we add the `milestone 2.8.0` label.

@Jennifer88huang Thanks for the explanation, and thank you for merging.

**Pull request merged**